index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ahsan Ghazni Khan's Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="">Ahsan Ghazni Khan</a>
+            <button><img src="images/Icon-Menu.png" alt=""></button>
+        </nav>
```

Comment:
- [ ] Great work introducing the `<nav>` tag here, although there's one thing that need to be corrected. Consider moving the ***logo (line 13)*** outside the `<nav>` tag